grumphp.yml

```diff
@@ -6,6 +6,19 @@ grumphp:
     failed: vendor/axelerant/drupal-quality-checker/resources/grumpy.txt
     succeeded: vendor/axelerant/drupal-quality-checker/resources/happy.txt
   tasks:
+    phplint:
+      exclude:
+        - web/core
+        - web/modules/contrib
+        - web/themes/contrib
+        - web/profiles/contrib
+        - web/sites/default
+        - vendor
+      triggered_by:
+        - php
+        - module
+        - theme
+        - inc
```

Comment:
This is duplicated.

Reply:
I think I messed it up while resolving merge conflict. I have removed the duplicate lines now.